CMakeLists.txt

```diff
@@ -15,8 +15,42 @@ list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/CMakeModules/Modules)
 ### App and Component Options
 ###############################################################################
 
+######################################
```

Comment:
I prefer not to do any big change in this file since it would make it hard for us to sync with ufs-weather-model. I know the list of applications is getting large but at this point we have one app for each component and this will be removed once we could able to compile all the components together. Please just add only your application to here line CSTLP or CSTLSP.